build.py

```diff
@@ -0,0 +1,42 @@
+import shutil
+from pathlib import Path
+
+from staticjinja import Site
+from yaml import Loader, load
+
+import sass
+
+
+def data():
+    yaml = Path("./data").glob("*.yaml")
+    data_dict = {
+        file_path.stem: load(file_path.read_text(), Loader=Loader) for file_path in yaml
+    }
+
+    return data_dict
+
+
+def clean_buildpath(output_dir):
+    if output_dir.exists() and output_dir.is_dir():
+        shutil.rmtree(output_dir)
+
+
+def main():
+    buildpath = Path("output")
```

Comment:
Could you get this as an optional CLI argument? This would allow passing the directory both in RTD and CI if necessary:
```suggestion
    try:
        build_dir_str = sys.argv[1]
    except IndexError:
        build_dir_str = "output"

    buildpath = Path(build_dir_str)
```

Reply:
Hmm. Yeah that could save a step in RTD. Cheers.